INVIOLABLES.md

```diff
@@ -81,7 +81,7 @@ Source: Oxford Dictionary of the English Language
 
 ## NuttX Branding
 
-  - The official name of authentic Nuttx will always be "NuttX".
+  - The official name of authentic NuttX will always be "NuttX".
```

Comment:
This should be updated to reflect Apache

Reply:
The trademark is NuttX, not Apache NuttX.  That name is not protected.

Reply:
But trademark aside this code is Apache NuttX not NuttX